open_event/api/custom_fields.py

```diff
@@ -133,3 +135,52 @@ def validate(self, value):
         except Exception:
             return False
         return True
+
+
+class String(CustomField):
+    """
+    Custom String Field
+    """
+    def validate(self, value):
+        if not value:
+            return self.validate_empty()
+        if value.__class__.__name__ in ['unicode', 'str']:
+            return True
+        else:
+            return False
+
+
+class Integer(CustomField):
+    """
+    Custom Integer Field
+    """
+    __schema_type__ = 'integer'
+    __schema_format__ = 'int'
+    __schema_example__ = 0
+
+    def validate(self, value):
+        if value is None:
+            return self.validate_empty()
+        if type(value) == int and value >= 0:
+            # No <0 int values right now
```

Comment:
@shivamMg 
Probably all the Integer that we will use have `>0` condition. So I didn't want to name every field as `PositiveInteger`. I have updated the code and added a param `positive`. `positive` is True by default, and can be made false if ever a negative integer field is needed.